tests/load/config.js

```diff
@@ -0,0 +1,15 @@
+import crypto from "k6/crypto";
+
+export const URL = "http://107.22.129.12";
```

Comment:
Consider adding a comment explaining this hard coded values  At minimum, can we specify that it is a place holder that needs to be replaced?